applications/asset_tracker_v2/src/modules/debug_module.c

```diff
@@ -33,6 +38,71 @@
 #include <zephyr/logging/log.h>
 LOG_MODULE_REGISTER(MODULE, CONFIG_DEBUG_MODULE_LOG_LEVEL);
 
+#if defined(CONFIG_NRF_MODEM_LIB_TRACE_BACKEND_RAM) && defined(CONFIG_MEMFAULT)
```

Comment:
With a new option that I proposed we could replace these options with the new one.